velox/functions/sparksql/Register.cpp

```diff
@@ -312,6 +313,8 @@ void registerFunctions(const std::string& prefix) {
       makeRepeatAllowNegativeCount,
       repeatMetadata());
 
+  registerArrayGetFunction(prefix + "get");
```

Comment:
Seems we can directly use `VELOX_REGISTER_VECTOR_FUNCTION(udf_array_get, name);` to register it. Then, no need to use `registerArrayGetFunction`.

Reply:
done, thank you!